### ISSUE TITLE
OTP 20.1

### ISSUE THREAD

**Issue by samphilipd** [asdf-vm/asdf-elixir#35]
I can't seem to install elixir 1.5.1 with OTP 20.1 - 

```
asdf install elixir 1.5.1-otp-20.1
/var/folders/3w/_r0kg92d567_gggr064_wjyh0000gn/T//elixir-precompiled-1.5.1-otp-20.1.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   243    0   243    0     0    519      0 --:--:-- --:--:-- --:--:--   520
[/var/folders/3w/_r0kg92d567_gggr064_wjyh0000gn/T//elixir-precompiled-1.5.1-otp-20.1.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /var/folders/3w/_r0kg92d567_gggr064_wjyh0000gn/T//elixir-precompiled-1.5.1-otp-20.1.zip or
        /var/folders/3w/_r0kg92d567_gggr064_wjyh0000gn/T//elixir-precompiled-1.5.1-otp-20.1.zip.zip, and cannot find /var/folders/3w/_r0kg92d567_gggr064_wjyh0000gn/T//elixir-precompiled-1.5.1-otp-20.1.zip.ZIP, period.
```

**Comment by ericmj** [comment]
You should only use the major OTP version. `asdf install elixir 1.5.1-otp-20`

**Comment by tgautier** [comment]
> You should only use the major OTP version. `asdf install elixir 1.5.1-otp-20`

I was going to 👍 the issue until I saw the comment from @ericmj 

Why shouldn't we use use minor updates from OTP?

**Comment by ericmj** [comment]
You can use minor updates from OTP, but with this command you are installing Elixir. We provide this Elixir build so you can install Elixir built for a given OTP version. We only build for OTP major versions because OTP guarantees compatibility for the entire major version so it's unnecessary to build for every OTP release.

**Comment by tgautier** [comment]
Thanks :)

**Comment by naps62** [comment]
I'm currently trying to fix an issue that seems to happen only on OTP 20, and is fixed in OTP 20.1 (source: https://bugs.erlang.org/browse/ERL-517)

Is there any way I can go around this?

**Comment by ericmj** [comment]
@naps62 `asdf_elixir` is for installing Elixir. If you want to install OTP 20.1, check out https://github.com/asdf-vm/asdf-erlang instead.

@samphilipd Please close this issue since there is no bug and the issue seems to confuse people.
